standardtracer/recorder.go

```diff
@@ -1,32 +1,9 @@
 package standardtracer
 
-// ProcessIdentifier is a thin interface that guarantees all implementors
-// represent a ProcessName and accepts arbitrary process-level tag assignment
-// (e.g., build numbers, platforms, hostnames, etc).
-type ProcessIdentifier interface {
```

Comment:
I removed this since a tracing impl can add this if it feels like it, but ProcessIdentifier has no bearing on RawSpan or its direct dependencies so thus can be omitted here.